rfc/rfc002-authentication-token.md

```diff
@@ -354,7 +359,7 @@ In order to validate an UZI signed JWS, the validator MUST perform the following
 The following snippets contain exaples of an UZI signed JWT with a login contract.
 
 {% code title="Header" %}
-```javascript
+```yaml
```

Comment:
i'd say `json`

Reply:
I believe that gitbook changes it to yaml if you edit from gitbook itself.